blank_project/src/config.js

```diff
@@ -30,8 +30,8 @@
             };
         case 'test':
             return {
```

Comment:
can we make `case 'test'` fallthrough to `case 'ci'` to void maintaining duplicate config?